docs/src/main/paradox/lagom/overview.md

```diff
@@ -0,0 +1,48 @@
+# Lagom gRPC support overview
+
+## gRPC
+
+[gRPC](https://grpc.io/) is a transport mechanism for request/response and (non-persistent) streaming use cases. See [Akka gRPC documentation for an explanation of why gRPC](https://developer.lightbend.com/docs/akka-grpc/current/whygrpc.html) and when to use it as your transport.
+
+## How to add the sbt plugin
+
+To get started, for both client and server code generation, you will need to add Akka gRPC sbt plugin:
+
+sbt
+:   @@@vars
+    ```scala
+    // in project/plugins.sbt:
+    addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "akka.grpc.version$")
+    ```
+    @@@ 
+
+
+And then in your `build.sbt` file, you will need to enable the plugin for the project that contains the `.proto` files:
+
+sbt
+:   @@@vars
+    ```scala
+    // in build.sbt:
+    enablePlugins(AkkaGrpcPlugin)
+    ```
+    @@@
```

Comment:
This looks weird in the rendered HTML. In Lagom, a project is always multimodule so enabling plugins globally is not common or recommended.